src/electron/adapters/electron-storage-adapter.ts

```diff
@@ -1,30 +1,35 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { IndexedDBDataKeys } from 'background/IndexedDBDataKeys';
 import { StorageAdapter } from 'common/browser-adapters/storage-adapter';
 import { IndexedDBAPI } from 'common/indexedDB/indexedDB';
-import { pick } from 'lodash';
 
 export class ElectronStorageAdapter implements StorageAdapter {
     constructor(private readonly indexedDBInstance: IndexedDBAPI) {}
 
     public async setUserData(items: Object): Promise<void> {
-        await this.indexedDBInstance.setItem(IndexedDBDataKeys.installation, items);
+        const setItemPromises = Object.keys(items).map(itemKey =>
+            this.indexedDBInstance.setItem(itemKey, items[itemKey]),
```

Comment:
I wasn't able to find a guarantee in the [docs](https://github.com/jakearchibald/idb-keyval#readme) for the underlying API we're using that it's okay to interleave `set` calls like this; maybe safer to await each setItem in turn 
rather than using `Promise.all`

Reply:
Good call. Updated.